### DISCUSSION TITLE
Sensor Battery versus % SoC start/end

### DISCUSSION THREAD

**Discussion by mff73** [mytourbook/mytourbook#1416] [Q&A]
Hello everyone,
long time without questions, here is one 😊
What is the difference between having % of battery levels at sensor level and  the activity data field %Soc Start and End ?
In my case, i am getting battery informations, from activity fit files (Suunto watch in my case), stored, and display like sensor data.
What should i do to get battery informations at activity display ?
![image](https://github.com/user-attachments/assets/21815722-fe18-433d-8114-0461555d7da0)

thanks, i am a bit lost in understanding what is what there :)

**Comment by wolfgang-ch** [comment]
The chart at the bottom shows the sensor values **"SUUNTO - Suunto Vertical"** but this is not 
the device value **"SUUNTO Suunto Vertical"** (without the dash)

Here you can see where the values are displayed

![image](https://github.com/user-attachments/assets/de4e9297-a0f2-481a-9750-bfa8823cccf2)

**Comment by mff73** [comment]
Thanks.
would there be a way for sensor values to become device values ?
in my case, the battery values from my device (suunto watch), included in the activity fit files, are considered as sensor values (you implemented this recently : https://github.com/mytourbook/mytourbook/discussions/1284). Sensor chart is working, but i would quite like to see SoC start/end in the tourbook as well, if possible of course.

**Comment by wolfgang-ch** [comment]
> would there be a way for sensor values to become device values ?

These values are total differently stored in the .fit file and this was the productor desicion how to store values in the .fit file and accordingly, these values are stored differently in a tour.

Device battery values are stored in the tour itself, sensor values, there can be many, are stored separately in it's own sensor table. To display the sensor values in the tourbook view, it needs an additional SQL Join and new columns in the tourbook view.
